spec/RedisCacheAdapter.spec.js

```diff
@@ -175,20 +175,20 @@ describe_only(() => {
 
   beforeEach(async () => {
     await cacheAdapter.clear();
-    getSpy = spyOn(cacheAdapter, 'get').and.callThrough();
-    putSpy = spyOn(cacheAdapter, 'put').and.callThrough();
     await reconfigureServer({
       cacheAdapter,
       enableSingleSchemaCache: true,
     });
+    getSpy = spyOn(cacheAdapter, 'get').and.callThrough();
```

Comment:
if you switch from fat arrow functions `() => {}` to `function ()` for all of all `desribe`, `it`, `beforeEach`, then you can `this.getSpy = spyOn(....`

And then use them in your tests like `expect(this.getSpy....`

not asking you to do it, just the patterns i use.